src/controller/controller.cpp

```diff
@@ -68,10 +68,10 @@ NWNXController::NWNXController(SimpleIniConfig* config)
 
 	if (!config->Read("nwn2", &nwnhome))
 	{
-		logger->Info("* NWN2 home directory not found. Check your nwnx.ini file.");
+		logger->Info("* NWN2 installation directory not found. Check your nwnx.ini file.");

```

Comment:
If the log changed, should we change the variable name as well for consistency?

Reply:
makes sense :)

FYI this name confused me a bit because the -home command-line arg of nwn2server sets the directory that is by default `Documents/Neverwinter Nights 2` ^^